react-form-controls/src/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class NewsLetterForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { value: undefined };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ value: event.target.value });
+  }
+
+  handleSubmit(event) {
+    console.log('An email was submitted: ' + this.state.value);
+    event.preventDefault();
+  }
+
+  render() {
+    return (
+      <div className = "container">
+        <form onSubmit={this.handleSubmit}>
+        <label>
+          Email:
+          <input type="text" onChange = {this.handleChange}/>
```

Comment:
The `value` attribute of the input needs to be set from state: `value={this.state.value}`